package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@google-cloud/pubsub",
   "description": "Cloud Pub/Sub Client Library for Node.js",
-  "version": "4.11.0",
+  "version": "5.0.0-rc.0",
```

Comment:
this will throw off release-please since it's supposed to handle upgrading this on its own. I know, sucks, means we have to release as a stable ontop of gax being an rc-candidate. Not ideal, we'll fix this for next upgrade.